storage/migrations/02_oasis_3_emerald.up.sql

```diff
@@ -91,6 +91,14 @@ CREATE TABLE oasis_3.address_preimages
     address_data       BYTEA NOT NULL
 );
 
+CREATE TABLE oasis_3.emerald_token_balances
+(
+  token_address TEXT NOT NULL,
+  account_address TEXT NOT NULL,
+  balance NUMERIC NOT NULL,
```

Comment:
Nit: `NUMERIC(1000,0)` to enforce integers; the 0 means no decimals.

Reply:
ok

Reply:
changed, will need further change since we since moved to types like uint_numeric

Reply:
#223 to track that